### PR TITLE
fix(select): clean up classes

### PR DESCRIPTION
**Pull request by felicia-haggqvist**
## Description
This PR ensures no classes styling the same CSS properties are being set on the same HTML element in the Select component.

Additionally this PR has:
-  Added missing props `disabled` and `readOnly` to Select component
- Added missing Select examples for `disabled` and `readOnly`.
- Added a `handleKeyDown()` function to disable `space`, `arrowUp` and `arrowDown` for when `readOnly` prop is set to `true`
- Added a more interactive invalid example (where the invalid state will be removed if the user chooses the right choice). 

## How to test
Either link this branch with @warp-ds/css (`fix/clean-up-select-classes` branch) or replace this for @warp-ds/css dependency in package.json: `github:warp-ds/css#component-classes-cleanup` with this  `github:warp-ds/css#fix/clean-up-select-classes`

### CODE REVIEW

File: packages/select/index.js [hunk 1]
```diff
@@ -36,29 +36,67 @@ export class WarpSelect extends kebabCaseAttributes(WarpElement) {
...
+
+    super.disconnectedCallback();
+  }
+
+  handleKeyDown(event) {
+    const target = event.target.closest('w-select[data-readonly-id="' + this._readOnlyId + '"]');
+    if (target && (event.key === ' ' || event.key === 'ArrowDown' || event.key === 'ArrowUp')) {
+      event.preventDefault();
+    }
+  }
```
Comment (BalbinaK) [inline packages/select/index.js:80]: Maybe we could utilise the `@keydown` event listener on the `select` element, like it was done in Card and Attention component. Then the `handleKeyDown" method could be more similar to what we have in React Select and the logic would be much less complicated :)

Comment (felicia-haggqvist) [reply packages/select/index.js:80]: That makes sense! I'll change it! 😄

Comment (felicia-haggqvist) [reply packages/select/index.js]: Updated in this commit: [0f706be](https://github.com/warp-ds/elements/pull/159/commits/0f706bee85360bd1ed77633b950f73f9f80f40d0)

**Review by BalbinaK — APPROVED** [review]
Looks great!

**Pull request merged**